dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GeneralWorkflowIT.java

```diff
@@ -1004,6 +1009,27 @@ public void testTestParameterFile() {
         final long count6 = testingPostgres
                 .runSelectStatement("select count(*) from sourcefile where type='WDL_TEST_JSON'", new ScalarHandler<>());
         Assert.assertEquals("there should be one sourcefile that is a wdl test parameter file, there are " + count6, 1, count6);
+        verifiedInformationMigrationTest(testingPostgres);
+    }
+
+    private void verifiedInformationMigrationTest(CommonTestUtilities.TestingPostgres testingPostgres) {
```

Comment:
As discussed, just copy this out into a new test so you don't need to immediately throw away the standard migration for this class. 
You can verify via database with a 1.4.0 schema and then run all of 1.5.0.
Should be simpler and you can simplify changes to CommonTestUtilities